README.md

```diff
@@ -1 +1,178 @@
-# toDoList
\ No newline at end of file
+![68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067](https://user-images.githubusercontent.com/106747387/230112602-9404e3e1-7fbb-4690-98e3-bbffbf5c314a.jpeg)
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List <a name="about-project"></a>
+
+This is a list of tasks that an individual needs to complete or accomplish. Tasks are typically put in order by priority or importance. 
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>html</li>
+    <li>css</li>
+    <li>javascript</li>
+    <li>webpack</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- *Add what to do*
+- *Remove after completed*
+- *Great UI*
+- 
+<!-- Live Demo -->
+
+### Live Demo <a name="live-demo"></a>
+
+[visit here](https://oluwatobi-fasasi.github.io/toDoList/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need the following tools:
+- Node.js in your computer
+- webpack
+- IDE(e.g.: Vscode,...)
+- Terminal
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  git clone https://github.com/oluwatobi-fasasi/To-Do-List
+```
+
+
+### Install
+
+- Install this project with:
+
+```sh
+  cd To-Do-List
+
+  npm install or npm i
+```
+
+- Run an optimized build for prodction:
+```
+  npm run build
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run dev
+
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm run test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@oluwatobi](https://github.com/oluwatobi-fasasi)
+- Twitter: [@Fasasi](https://twitter.com/FasasiTobi1)
+- LinkedIn: [Fasasi](https://www.linkedin.com/in/fasasi-abdulwasih-oluwatobi-129a8b109/)
+
```

Comment:
![image](https://github.com/oluwatobi-fasasi/toDoList/assets/93039359/2eb01305-dd9a-4b04-826d-3a6644e82aa4)

- [ ] It would be nice if you fix your LinkedIn account handle because right now it is broken because it is giving me 404 error which you can see in the image above ⬆️ Kindly fix this error to make your readme descriptive and professional 👍